src/pages/invoices/invoiceDetails.js

```diff
@@ -0,0 +1,177 @@
+import { Button, CssBaseline, Box, Typography, Divider } from '@mui/material';
+import data from '../../components/InvoiceDetailsDatabase';
+import { ArrowBack, ArrowForwardIos } from '@mui/icons-material';
+import { useRouter } from 'next/router';
+
+const Details = () => {
+  const router = useRouter();
+  const invoice = data.filter(
+    (element) => element.id === Number(router.query.id)
+  )[0];
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        maxWidth: '100vw',
+        minHeight: '100vh',
+        m: '0 auto',
+      }}
+    >
+      <CssBaseline />
+      <Box sx={{ maxHeight: '70%' }}>
+        <Box
+          display="flex"
+          alignItems="center"
+          color="#32424E"
+          sx={{ p: '.5em' }}
+        >
+          <ArrowBack onClick={() => router.back()} />
+          <Typography variant="h5" sx={{ fontWeight: '600', ml: '.75em' }}>
+            October
+          </Typography>
+        </Box>
+        <Box
+          sx={{
+            p: '0 1em',
+            color: '#fff',
+            background: 'linear-gradient(90deg, #26A1B7 0%, #0372AE 100%)',
+          }}
+        >
+          <Box
+            display="flex"
+            justifyContent="space-between"
+            sx={{ maxWidth: '40em', m: '0 auto' }}
+          >
+            <Box sx={{ color: '#fff' }}>
+              <Typography variant="caption">Invoice number</Typography>
+              <Typography variant="body1">{invoice.id}</Typography>
```

Comment:
```suggestion
              <Typography>{invoice.id}</Typography>
```

The [docs](https://mui.com/api/typography/) mention `body1` as being the default value of the `variant` prop of `Typography` components, so in this case it can be safely removed.  Check for other places where this applies.